X/XNNPACK/bundled/patches/xnnpack-soversion.patch

```diff
@@ -12,4 +12,4 @@ index 9bcaa5816..dbe149f85 100644
 +  SOVERSION 0)
  IF(CMAKE_SYSTEM_PROCESSOR MATCHES "^armv[5-8]" OR IOS_ARCH MATCHES "^armv7")
    SET_PROPERTY(SOURCE ${XNNPACK_MICROKERNEL_SRCS} APPEND_STRING PROPERTY COMPILE_FLAGS " -marm ")
-   SET_PROPERTY(SOURCE ${XNNPACK_PSIMD_FASTMATH_MICROKERNEL_SRCS} ${XNNPACK_PSIMD_ACCMATH_MICROKERNEL_SRCS} APPEND_STRING PROPERTY COMPILE_FLAGS " -mfpu=neon ")
+   SET_PROPERTY(SOURCE ${XNNPACK_NEON_MICROKERNEL_SRCS} APPEND_STRING PROPERTY COMPILE_FLAGS " -march=armv7-a -mfpu=neon ")
```

Comment:
Why is this necessary?

Reply:
I just updated the patches, so they (should apply) applied more nicely.

Reply:
PSIMD was removed as a dependency of XNNPACK.

Reply:
I was more specifically referring to `-march=armv7-a`.

Reply:
Those lines are just the context of the patch: https://github.com/google/XNNPACK/blob/79cd5f9e18ad0925ac9a050b00ea5a36230072db/CMakeLists.txt#L3473

I was just trying to edit as little as possible in the patches, so this one only inserts a VERSION and SOVERSION: https://github.com/google/XNNPACK/blob/79cd5f9e18ad0925ac9a050b00ea5a36230072db/CMakeLists.txt#L3470

Reply:
... the `march` arguments is being removed by https://github.com/JuliaPackaging/Yggdrasil/pull/5416/files#diff-8eb5c3a4ae80bba315cf237677664f0c81196496be3bf80c10a394d64c9e5563

Reply:
Reading the diff of a diff is hard :weary:

Reply:
Indeed :-)

Upstreaming patches...